tests/unit/assert_test.sh

```diff
@@ -109,16 +109,37 @@ function test_unsuccessful_assert_not_empty() {
     "$(assert_not_empty "")"
 }
 
+function test_successful_assert_not_same() {
+  assert_empty "$(assert_not_same "1" "2")"
+}
+
+function test_unsuccessful_assert_not_same() {
+  assert_same\
+    "$(console_results::print_failed_test "Unsuccessful assert not same" "1" "but got " "1")"\
+    "$(assert_not_same "1" "1")"
+}
+
 function test_successful_assert_not_equals() {
   assert_empty "$(assert_not_equals "1" "2")"
 }
 
 function test_unsuccessful_assert_not_equals() {
-  assert_same\
```

Comment:
Cannot we use assert_same here? 🤔

Reply:
@Chemaclass 
Thank you, This line did not need to be modified. so, I will revert it now.